tests.js

```diff
@@ -614,6 +680,150 @@ window.Specs = {
 		}
 	},
 
+	"css2-selectors": {
+		"title": "CSS 2 Selectors",
+		"links": {
+			"tr": "CSS22/selectors.html",
+			"dev": "css2/"
+		},
+		"status": {
+			"stability": "stable",
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"selectors": {
+			"Universal selector": {
+				"links": {
+					"tr": "#universal-selector",
+					"dev": "#universal-selector"
+				},
+				"tests": "*"
+			},
+			"Type selector": {
+				"links": {
+					"tr": "#type-selectors",
+					"dev": "#type-selectors"
+				},
+				"tests": "h1"
+			},
+			"Descendant selector": {
+				"links": {
+					"tr": "#descendant-selectors",
+					"dev": "#descendant-selectors"
+				},
+				"tests": "div p"
+			},
+			"Child selector": {
+				"links": {
+					"tr": "#child-selectors",
+					"dev": "#child-selectors"
+				},
+				"tests": "div > p"
+			},
+			"Adjacent sibling selector": {
+				"links": {
+					"tr": "#adjacent-selectors",
+					"dev": "#adjacent-selectors"
+				},
+				"tests": "div > p"
+			},
+			"Attribute selectors": {
+				"links": {
+					"tr": "#attribute-selectors",
+					"dev": "#attribute-selectors"
+				},
+				"tests": [
+					"[title]", "[title=example]", "[title=\"example\"]", "[rel~=copyright]",
+					"[rel~=\"copyright\"]", "[lang|=en]", "[lang|=\"en\"]"
+				]
+			},
+			"ID selector": {
+				"links": {
+					"tr": "#id-selectors",
+					"dev": "#id-selectors"
+				},
+				"tests": "#id"
+			},
```

Comment:
We could also test the class selector: `.class`

Reply:
Yes. Added a test for it.